_includes/scenarios/integrate/create-attribute-for-yourself.md

```diff
@@ -0,0 +1,99 @@
+
+Since knowledge about IdentityAttributes is required in the following, you should take a look at our IdentityAttribute introduction before you continue reading this guide. In particular, a description of the two kinds of IdentityAttributes, the simple IdentityAttributes and the complex IdentityAttributes, can be found there.
+{: .notice--info}
+
+<!--- TODO: Insert link to "IdentityAttribute introduction" --->
+
+### Input for creating a RepositoryAttribute
+
+To create a RepositoryAttribute, proceed as described in the [Create a RepositoryAttribute]({% link _docs_use-cases/use-case-consumption-create-a-repositoryattribute.md %}) use case documentation. As input for the creation of a RepositoryAttribute, the following `content` must be used:
+
+```jsonc
+{
+  "content": {
+    "validFrom": "<start of IdentityAttribute's validity>",
+    "validTo": "<end of IdentityAttribute's validity>",
+    "value": {
+      // IdentityAttributeValue
+      ...
+    },
+    "tags": ["<additional information 1>", ..., "<additional information n>"]
+  }
+}
+```
+
+You need to replace the placeholders marked with `<...>` appropriately. Also, it is necessary that you insert one of the available [IdentityAttributeValues]({% link _docs_integrate/attribute-values.md %}#identity-attributes) into the `value` property. Note that the properties `validFrom`, `validTo` and `tags` are optional, so you can omit them.
+
+### Process of creating a RepositoryAttribute
+
+As you can see from the diagram below, after you have entered the [input for creating a RepositoryAttribute]({% link _docs_integrate/create-attribute-for-yourself.md %}#input-for-creating-a-repositoryattribute), a check is performed whether the input values for the properties of the specified [IdentityAttributeValue]({% link _docs_integrate/attribute-values.md %}#identity-attributes) meet the validation criteria documented on the [Attribute Values]({% link _docs_integrate/attribute-values.md %}) page. If the validation is not successful, an [error message]({% link _docs_integrate/error-codes.md %}) is sent in response. Otherwise, a RepositoryAttribute is created that contains the IdentityAttribute in its `content` property. If it is a simple IdentityAttribute, a success response is sent directly. In the case of a complex IdentityAttribute, on the other hand, another RepositoryAttribute is created beforehand for each of its appropriate components. These RepositoryAttributes for the components are also referred to as children of the RepositoryAttribute belonging to the complex IdentityAttribute. The `id` of their parent is contained within their `parentId` property.
```

Comment:
"components" replaced by "properties". 👍

Regarding the term "child Attribute", I would prefer not to use it in the documentation, and simply use "child" instead, so that it has less neologism vibes.

Reply:
I must admit that I don't see your "neologism vibes" point, since having an Attribute that is referred to as "child" isn't too far from calling it a "child Attribute", but I don't mind if you just call them "children".